src/main/java/org/junit/rules/ExternalResource.java

```diff
@@ -77,8 +77,10 @@ protected void before() throws Throwable {
 
     /**
      * Override to tear down your specific external resource.
+     *
+     * @throws Exception if teardown fails
      */
-    protected void after() {
+    protected void after() throws Exception {
```

Comment:
Just because AFAIK rarely methods throw Throwable, it's more likely they throw Exception, e.g. `AutoCloseable#close`.
But ya, we should probably make it more generic and throw Throwable.
Do you want me to?